pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -723,13 +725,19 @@ func (r *ReconcileHyperConverged) ensureKubeVirtConfig(req *hcoRequest) error {
 	}
 	objectreferencesv1.SetObjectReference(&req.instance.Status.RelatedObjects, *objectRef)
 
-	if !reflect.DeepEqual(found.Data, kubevirtConfig.Data) {
-		req.logger.Info("Updating existing KubeVirt config")
-		found.Data = kubevirtConfig.Data
-		err = r.client.Update(req.ctx, found)
-		if err != nil {
-			req.logger.Error(err, "Failed updating an existing kubevirt config")
-			return err
+	if r.upgradeMode {
+		// only virtconfig.SmbiosConfigKey, virtconfig.MachineTypeKey and selinuxLauncherType are
+		// going to be manipulated and only on HCO upgrades
+		for _, k := range []string{virtconfig.SmbiosConfigKey, virtconfig.MachineTypeKey, "selinuxLauncherType"} {
```

Comment:
```suggestion
		for _, k := range []string{virtconfig.SmbiosConfigKey, virtconfig.MachineTypeKey, virtconfig.SELinuxLauncherTypeKey} {
```
why not use `virtconfig.SELinuxLauncherTypeKey` like other two?